src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,14 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="page-content">
+    <div className="movies">
+      {
+        movies && movies.map(movie => (
```

Comment:
Just don't render this component if there are no movies 
```suggestion
      {movies.map(movie => (
```